onnx/defs/schema.h

```diff
@@ -12,16 +12,18 @@
 #include <ostream>
 #include <set>
 #include <string>
-#include <tuple>
 #include <unordered_map>
 #include <unordered_set>
 #include <vector>
 
 #include "data_type_utils.h"
+#include "onnx/checker.h"
```

Comment:
what do you need from checker in schema? this will introduce circular dependencies.

Reply:
Fixed.